syntax = "proto3";
package SvrData;
